src/Voron/Impl/Backup/FullBackup.cs

```diff
@@ -167,6 +167,32 @@ public class StorageEnvironmentInformation
 
                 try
                 {
+                    if (env.Options is StorageEnvironmentOptions.DirectoryStorageEnvironmentOptions)
+                    {
+                        foreach (var recoveryFile in Directory.GetFiles(env.Options.BasePath.FullPath, "*.Recovery"))
```

Comment:
Why are we taking those files now?

Reply:
So recovery of database snapshot could work